src/main/java/com/wcc/platform/domain/platform/Member.java

```diff
@@ -45,7 +49,7 @@ public Member(
     this.network = network;
   }
 
-    public Member() {
-        // Necessary constructor for jackson.
-    }
+  public Member() {
```

Comment:
you can remove this and use lombok: 

@AllArgsConstructor
@NoArgsConstructor 

Please make sure you have the latest changes merged.